src/reporters/FixReporter.ts

```diff
@@ -26,8 +27,7 @@ export class FixReporter implements IReporter {
     practicesAndComponents: PracticeWithContextForReporter[],
     practicesAndComponentsAfterFix: PracticeWithContextForReporter[],
   ): Promise<void> {
-    const reportString = this.buildReport(practicesAndComponents, practicesAndComponentsAfterFix);
-    console.log(reportString);
+    debug.log(this.buildReport(practicesAndComponents, practicesAndComponentsAfterFix));
```

Comment:
`debugLog `?

Reply:
`debugLog()` doesn't log in terminal. `debug.log()` logs both in the terminal and in the file if the `-l` is used.